packages/headless/src/database/mongodb/branch/index.ts

```diff
@@ -20,25 +27,35 @@ export class BranchDatabase implements IBranchDatabase {
         }
     };
 
+    async getURL(url: string): Promise<boolean> {
+      try {
+        const urlExists = await BranchModel.findOne({ url });
```

Comment:
getURL & getStore functions are the same strategy. So you may check getStore by passing url in service layer.